client/src/components/common/Forms/BasicForm/Forms/CustoForms/ResistrationForm.tsx

```diff
@@ -1,46 +1,60 @@
 import InputItem from "../../../../Input/InputItem";
 import BasicSelect from "../../../../../Select/Select";
 import { selectRoleColumns } from "../../../../../../config/config";
+import { FormControl, FormHelperText, Button } from "@mui/material";
+import styled from "styled-components";
 
-const RegistrationForm = ({ onChange, columns, userId, submitForm }: any) => {
+const StyledButton = styled(Button)`
```

Comment:
use Button property color, you can set default primary color in theme

https://mui.com/material-ui/api/button/